topics/icrc_25_signer_interaction_standard.md

```diff
@@ -16,13 +16,17 @@ This specification describes a communication protocol between dapps (decentraliz
 
 ## Transport Requirements
 
-This standard is agnostic to the transport mechanism / channel used to send the messages, as long as it provides authenticity and integrity. Confidentiality is not required.
+This standard is agnostic to the transport channel used to send the messages, as long as it provides the following properties:
+* The channel needs to provide authenticity and integrity: this means that the communicating parties know the other participant and can be sure that the messages they receive are sent by the party they expect and that the messages have not been tampered with.
+* The channel needs to be able to provide sessions: This means that channels needs to provide a way to establish a session between the signer and the relying party. The session must be established before the first message is sent. The default session length should be appropriate for the expected duration of the interaction between the relying party and the signer. The users must be able to explicitly terminate the session, and it must be terminated automatically after a certain period of inactivity. The session might be extended automatically if the interaction between the relying party and the signer is still _actively_ ongoing when the default session timeout is reached. There must be a maximum session duration (regardless of activity). Permission scopes (see [`permission` message](#permission)) are granted for the duration of a single session only.
```

Comment:
Hmm, the entire sessions-requirement seems somewhat "contrived", i.e. it lists several properties of sessions, and feels overly specific and too long.  Could the required property be somehow abstracted and shortened?
I see that it already does try to be more abstract than the recommendation in the finding (which talks about TLS certificates), but it still doesn't feel to be spot-on. 🤔

Reply:
As discussed, made the session a concept of the standard itself rather than being madated by the transport channel.